crates/node/src/validate.rs

```diff
@@ -110,6 +154,33 @@ pub async fn validate(
     conn_pool: &ConnectionPool,
     contract_registry: &ContentAddress,
     block: &Block,
+) -> Result<ValidateOutcome, ValidationError> {
+    let dry_run = DryRun::new(conn_pool.clone(), block).await?;
+    let db_type = Db::DryRun(dry_run);
+    validate_inner(db_type, contract_registry, block).await
+}
```

Comment:
It looks like this makes the `validate` behaviour the dry-run behaviour - should we name this `validate_dry_run` for clarity, or maybe update the docs?